Lib/tkinter/font.py

```diff
@@ -118,6 +118,11 @@ def __del__(self):
         except Exception:
             pass
 
+    def __neg__(self):
+        cp = self.copy()
+        cp['size'] = -(cp['size'])
```

Comment:
Parentheses are not necessary here: indexing/slicing is higher precedence than unary negation: see the [operator precedence table](https://docs.python.org/3/reference/expressions.html#operator-precedence).